Scripts/Services/HuntmasterChallenge/HuntMasterRewardGump.cs

```diff
@@ -54,13 +54,56 @@ public override void RemovePoints(double points)
             }
         }
 
+        public override void OnConfirmed(CollectionItem citem, int index)
+        {
+            if (citem.Type == typeof(RecipeScroll))
+            {
+                Item item;
+
+                switch (citem.Tooltip)
+                {
+                    default:
+                    case 1158769: item = new RecipeScroll(605); break;
+                    case 1158770: item = new RecipeScroll(606); break;
+                    case 1158771: item = new RecipeScroll(607); break;
+                }
+
+                if (item != null)
+                {
+                    if (User.Backpack == null || !User.Backpack.TryDropItem(User, item, false))
+                    {
+                        User.SendLocalizedMessage(1074361); // The reward could not be given.  Make sure you have room in your pack.
+                        item.Delete();
+                    }
+                    else
+                    {
+                        OnItemCreated(item);
+
+                        User.SendLocalizedMessage(1073621); // Your reward has been placed in your backpack.
+                        RemovePoints(citem.Points);
+                        User.PlaySound(0x5A8);
+                    }
+                }
+            }
+            else
+            {
+                base.OnConfirmed(citem, index);
+            }
+        }
+
         private static List<CollectionItem> _Collections = new List<CollectionItem>
         {
             new CollectionItem(typeof(HarvestersBlade), 0x2D20, 1114096, 0, 1.0),
             new CollectionItem(typeof(HornOfPlenty), 18080, 1153503, 0, 1.0),
             new CollectionItem(typeof(HuntmastersRewardTitleDeed), 0x14EF, 0, 0, 1.0),
             new CollectionItem(typeof(RangersGuildSash), 0x1541, 0, 0, 1.0),
             new CollectionItem(typeof(GargishRangersGuildSash), 0x46B5, 0, 0, 1.0),
+            new CollectionItem(typeof(RecipeScroll), 0x2831, 1158769, 0, 1.0), // hamburger recipe
+            new CollectionItem(typeof(RecipeScroll), 0x2831, 1158770, 0, 1.0), // hot dog recipe
+            new CollectionItem(typeof(RecipeScroll), 0x2831, 1158771, 0, 1.0), // sausage recipe
+            new CollectionItem(typeof(MinersSatchel), 0xA272, 1158773, 0, 1.0),
+            new CollectionItem(typeof(LumbjacksSatchel), 0xA274, 1158772, 0, 1.0),
```

Comment:
Errors:
 + Services/HuntmasterChallenge/HuntMasterRewardGump.cs:
    CS0246: Line 104: The type or namespace name 'MinersSatchel' could not be fo
und (are you missing a using directive or an assembly reference?)
    CS0246: Line 105: The type or namespace name 'LumbjacksSatchel' could not be
 found (are you missing a using directive or an assembly reference?)
Scripts: One or more scripts failed to compile or no script files were found.